src/bindings/python/src/openvino/opset16/ops.py

```diff
@@ -4,7 +4,7 @@
 
 """Factory functions for ops added to openvino opset16."""
 from functools import partial
-from typing import Optional
+from typing import Optional, Union
```

Comment:
```suggestion
from typing import Optional
```
Based on change Union is not needed and may be removed